qemu/tests/sev_hotplug_mem.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+from avocado import TestError
+from virttest import error_context
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+class MemoryHotplugSimple(MemoryHotplugTest):
+
+    def get_mem_by_name(self, vm, name):
+        """
+        Return memory object and pc-dimm devices by given name
+        """
+        dev_ids = map(lambda x: "-".join([x, name]), ["mem", "dimm"])
+        devices = filter(None, map(vm.devices.get_by_qid, dev_ids))
+        return [_[0] for _ in devices]
+
+    def unplug_memory(self, vm, target_mem):
```

Comment:
It looks to me that these two functions are not required for your testing, what you need to do might be:

```
if target_mem in vm.params.objects('mem_devs'):
        hotplug_test.unplug_memory(vm, target_mem)
else:
        hotplug_test.hotplug_memory(vm, target_mem)
hotplug_test.check_memory(vm)
```

Reply:
Thanks zhenchao, this looks more logical to me. Just remove the following codes as all  mem devices need to be unplugged is in plugged[].
for target_mem in plugged:
    hotplug_test.unplug_memory(vm, target_mem)
    hotplug_test.check_memory(vm)